test/fixtures/yarn-pnp/README.md

```diff
@@ -0,0 +1,17 @@
+# Yarn PnP
+
+This folder contains an example single-root workspace used as a test fixture for
+the Biome extension. The workspace folder is setup as a yarn pnp project.
```

Comment:
If this is an example folder, wouldn't having these files in an `examples/` folder be more coherent?

Reply:
I've used the same template as other fixtures for consistency. I can definitely change it, but then it would be inconsistent. What do you think?